pkg/reconciler/intevents/topic/topic.go

```diff
@@ -107,6 +108,12 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, topic *v1alpha1.Topic) r
 	// Set the topic being used.
 	topic.Status.TopicID = topic.Spec.Topic
 
+	// If the Topic was created from a Source, then we don't need to create the Publisher.
+	if _, ok := topic.Labels[SourceLabelKey]; ok {
```

Comment:
My preference is to use a spec field to do this, rather than introspect the labels. Something like `spec.publisher`, which defaults to `true`, but will be set to `false` when creating Topics for sources.